third_party/device_calendar/android/src/main/kotlin/com/builttoroam/devicecalendar/CalendarDelegate.kt

```
@@ -0,0 +1,1005 @@
+            GlobalScope.launch(Dispatchers.IO + exceptionHandler) {
+                while (eventsCursor?.moveToNext() == true) {
+                    val event = parseEvent(calendarId, eventsCursor) ?: continue
+                    events.add(event)
+                }
+                for (event in events) {
+                    val attendees = retrieveAttendees(calendar, event.eventId!!, contentResolver)
+                    event.organizer = attendees.firstOrNull { it.isOrganizer != null && it.isOrganizer }
+                    event.attendees = attendees
+                    event.reminders = retrieveReminders(event.eventId!!, contentResolver)
+                }
+            }.invokeOnCompletion { cause ->
```

detekt (1.23.8)

[warning] This use of GlobalScope should be replaced by `CoroutineScope` or `coroutineScope`.

(detekt.coroutines.GlobalCoroutineUsage)